packages/table/src/table.tsx

```diff
@@ -1156,6 +1156,12 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         e.stopPropagation();
 
         const { focusedCell } = this.state;
+        if (focusedCell == null) {
```

Comment:
Hm, what interaction made this necessary? We should look into how we want to support this, if we do. Which we may not.